go/test/endtoend/clustertest/vtctld_test.go

```diff
@@ -48,40 +46,15 @@ func TestVtctldProcess(t *testing.T) {
 	url := fmt.Sprintf("http://%s:%d/api/keyspaces/", clusterInstance.Hostname, clusterInstance.VtctldHTTPPort)
 	testURL(t, url, "keyspace url")
 
-	healthCheckURL := fmt.Sprintf("http://%s:%d/debug/health/", clusterInstance.Hostname, clusterInstance.VtctldHTTPPort)
+	healthCheckURL := fmt.Sprintf("http://%s:%d/debug/health", clusterInstance.Hostname, clusterInstance.VtctldHTTPPort)
 	testURL(t, healthCheckURL, "vtctld health check url")
 
-	url = fmt.Sprintf("http://%s:%d/api/topodata/", clusterInstance.Hostname, clusterInstance.VtctldHTTPPort)
```

Comment:
Did we remove this API?

Reply:
Looks like it was part of the `vtctld/explorer`? See https://github.com/vitessio/vitess/pull/11851/files#diff-8aca76c457b3084aef2637ba1c94220414367a86629e3d24a5e160ed0fbeda14L170-L172

I wonder if it was in the wrong place and should be moved to `vtctld/api` then instead?

Reply:
I did remove it because I thought it served just the old topology browser - but I think you're right, since we're keeping the API we should keep this part too.

Reply:
Added it back!
![Screenshot 2023-01-20 at 12 49 08 PM](https://user-images.githubusercontent.com/31225471/213802542-728493ed-9378-4efc-9e67-cff7b5724894.png)